x-pack/plugin/inference/qa/test-service-plugin/src/main/java/org/elasticsearch/xpack/inference/mock/AbstractTestInferenceService.java

```diff
@@ -27,14 +27,6 @@
 
 public abstract class AbstractTestInferenceService implements InferenceService {
 
-    protected static int stringWeight(String input, int position) {
```

Comment:
Moved to specific methods on the different dense and sparse vector mock services